src/openrct2/peep/Peep.h

```diff
@@ -635,7 +635,7 @@ struct Peep : SpriteBase
     };
     ride_id_t InteractionRideIndex;
     uint16_t TimeInQueue;
-    uint8_t RidesBeenOn[32];
+    ride_id_t RidesBeenOn[32];
```

Comment:
Port to develop.

Reply:
This is going to be a breaking change. I don't think it can be ported

Reply:
Hm, that's annoying. I guess there are no peep changes planned for the immediate future?